graphistry/umap_utils.py

```diff
@@ -165,7 +165,7 @@ def umap(
         self,
         kind: str = "nodes",
         use_columns: Union[List, None] = None,
-        featurize: bool = True,
+        featurize: bool = False,
```

Comment:
@silkspace  is  disabling by default intentional? 

my thinking is default should be default on, and if libs not installed, we nudge them to install or disable. I can imagine an `auto` mode that tries to do the best for available installed packages, but seems not worth it yet..